pkg/metrics/collector.go

```diff
@@ -566,9 +566,6 @@ func (c ResourcesMetricsCollector) collectVulnerabilityIdReports(ctx context.Con
 				}
 				var vulnList = make(map[string]bool)
```

Comment:
@alexanderwoehler looks like vulnList var should be removed since it was only used to track vuln IDs that were already found